config/i18n-tasks.yml

```diff
@@ -288,6 +288,9 @@ ignore_unused:
   - decidim.debates.debate_m.commented_time_ago
   - decidim.elections.trustee_zone.trustees.show.identification_keys.upload_error.*
   - decidim.elections.feedback.answer.*
+  - decidim.votings.admin_log.voting.create
```

Comment:
I think this is better, but all the other modules add all the translations at once, using `decidim.[module_name].admin_log.*`. So, maybe we should use the same approach, what do you think?